lib/action.js

```diff
@@ -90,6 +91,28 @@ export class Action {
         return global.display.get_tab_list(0, active_workspace)
     }
 
+    get_window_cmdline(window) {
+        const pid = window.get_pid();
+
+        if (!pid) return "";
+
+        const file = Gio.File.new_for_path(`/proc/${pid}/cmdline`);
+        const [ok, contents] = file.load_contents(null);
+        if (!ok) return "";
+
+        try {
+            const file = Gio.File.new_for_path(`/proc/${pid}/cmdline`);
```

Comment:
These two lines (which might be perfomance-inefficient) are run twice. Is it on purpose?